modules/islandora_scholar_embargo/includes/admin.form.inc

```diff
@@ -21,7 +21,9 @@ function islandora_embargo_admin(array $form, array &$form_state) {
   $options = islandora_get_content_models();
   $selected = variable_get('islandora_embargo_content_models', array('ir:citationCModel', 'ir:thesisCModel'));
   foreach ($selected as $cmodel) {
-    $options = array($cmodel => $options[$cmodel]) + $options;
+    if (isset($options[$cmodel])) {
+      $options = array($cmodel => $options[$cmodel]) + $options;
+    }
```

Comment:
It's just because it's possible for those options not to be returned in islandora_get_content_models() due to XACML or other what-have-ye's, so $options[$cmodel] may not exist. But yeah, there's probably a more PHP-weird-array-business way to do it?